code/Controllers/CMSPageHistoryController.php

```diff
@@ -159,7 +159,7 @@ public function EditForm($request = null)
      * Returns the read only version of the edit form. Detaches all {@link FormAction}
      * instances attached since only action relates to revert.
      *
-     * Permission checking is done at the {@link CMSMain::getEditForm()} level.
+     * Permission checking is done at the {@link SilverStripe\CMS\Controllers\CMSMain::getEditForm()} level.
```

Comment:
Did we want to have the FQN all over the docs or perhaps we could just `use SilverStripe\CMS\Controllers\CMSMain` at the top?